test/app/features/profile/data/repositories/user_profile_repository_test.dart

```diff
@@ -1,60 +1,75 @@
 
-import '../../../../../utils/helper.mocks.dart';
+class MockApiProvider extends Mock implements IApiProvider {}
+
+class MockApiServerConfigure extends Mock implements IApiServerConfigure {}
 
 void main() {
-  late final MockIApiProvider apiProvider = MockIApiProvider();
-  late final MockIApiServerConfigure serverConfiguration =
-      MockIApiServerConfigure();
-  late final IUserProfileRepository sut = UserProfileRepository(
-    apiProvider: apiProvider,
-    serverConfiguration: serverConfiguration,
-  );
+  late IUserProfileRepository sut;
+  late IApiProvider apiProvider;
+  late IApiServerConfigure serverConfiguration;
+
+  setUpAll(() {
+    apiProvider = MockApiProvider();
+    serverConfiguration = MockApiServerConfigure();
+    sut = UserProfileRepository(
+      apiProvider: apiProvider,
+      serverConfiguration: serverConfiguration,
+    );
+  });
 
   void _setUpMockPost() {
     when(
-      apiProvider.post(
-        path: anyNamed('path'),
-        parameters: anyNamed('parameters'),
+      () => apiProvider.post(
+        path: any(named: 'path'),
+        parameters: any(named: 'parameters'),
       ),
     ).thenAnswer((_) async => Future.value(''));
   }
 
-  group('UserProfileRepository', () {
+  group(UserProfileRepository, () {
     group('stealth mode', () {
-      test('should hit endpoint /me/modo-camuflado-toggle', () async {
-        // arrange
-        _setUpMockPost();
-        final endPoint = ['me', 'modo-camuflado-toggle'].join('/');
-        final parameters = {'active': '1'};
-        // act
-        await sut.stealthMode(toggle: true);
-        // assert
-        verify(
-          apiProvider.post(
-            path: endPoint,
-            parameters: parameters,
-          ),
-        );
-      });
-      test('should receive ValidField', () async {
-        // arrange
-        _setUpMockPost();
-        final actual = right(const ValidField());
-        const toggle = true;
-        // act
-        final expected = await sut.stealthMode(toggle: toggle);
-        // assert
-        expect(actual, expected);
-      });
+      test(
+        'hit endpoint /me/modo-camuflado-toggle',
+        () async {
+          // arrange
+          _setUpMockPost();
+          final endPoint = ['me', 'modo-camuflado-toggle'].join('/');
+          final parameters = {'active': '1'};
+          // act
+          await sut.stealthMode(toggle: true);
+          // assert
+          verify(
+            () => apiProvider.post(
+              path: endPoint,
+              parameters: parameters,
+            ),
+          ).called(1);
+        },
+      );
+      test(
+        'receive ValidField',
+        () async {
+          // arrange
+          _setUpMockPost();
+          final actual = right(const ValidField());
+          const toggle = true;
+          // act
+          final expected = await sut.stealthMode(toggle: toggle);
+          // assert
+          expect(actual, expected);
```

Comment:
Sei que no caso foi mantida a estrutura dos testes, porém poderia aproveitar e ajustar o nome das variáveis? Pois caso o teste falhe, pode ficar um pouco confuso de se entender a mensagem sendo que apareceria erroneamente que o retorno de `stealthMode` é o esperado.

```suggestion
          final expected = right(const ValidField());
          const toggle = true;
          // act
          final actual = await sut.stealthMode(toggle: toggle);
          // assert
          expect(actual, expected);
```